src/node.cc

```diff
@@ -2843,6 +2844,11 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_print_eval", True(env->isolate()));
   }
 
+  // -c, --check
+  if (syntax_check_only) {
+    READONLY_PROPERTY(process, "_syntax_check_only", True(env->isolate()));
```

Comment:
I'd be happy to change this if everyone agrees this should be done - I just tried to match the style I saw in this file.